src/GitHub/Endpoints/Repos/Traffic.hs

```diff
@@ -0,0 +1,73 @@
+-- | The traffic API, as described at <https://developer.github.com/v3/repos/traffic/>
+module GitHub.Endpoints.Repos.Traffic (
+    popularReferrers',
+    popularReferrersR,
+    popularPaths',
+    popularPathsR,
+    views',
+    viewsR,
+    clones',
+    clonesR
+    ) where
+
+import Data.Vector (Vector)
+
+import GitHub.Data
+       (Auth, Clones, Error, Name, Owner, Period, PopularPath, Referrer, Repo,
+       Views, prettyPeriod)
+import GitHub.Data.Request (query, toPathPart)
+import GitHub.Request      (Request, executeRequest)
+
+-- | The top 10 referrers for the past 14 days.
+--
+-- > popularReferrers' (BasicAuth "github-username" "github-password") "qfpl" "tasty-hedgehog"
```

Comment:
Hmm, `tasty-hedgehog` promotion?

Reply:
Just the repo I was using to test at the time. Happy to change it.

Reply:
It's fine, it reminds people who contributed this module :)